api/v3/utils.php

```diff
@@ -1798,6 +1798,9 @@ function _civicrm_api_get_fields($entity, $unique = FALSE, &$params = array()) {
     }
   }
   $fields += _civicrm_api_get_custom_fields($entity, $params);
+  if ($entity == 'Contact') {
+    $fields += _civicrm_api_get_fields('Address');
+  }
```

Comment:
Would be nice to not place hacks for a specific entity in the generic api layer (any more than there already are).
Shouldn't this be in _civicrm_api3_contact_get_spec?